operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,110 @@
+# Audit Logs
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+By default, host and Kubernetes API audit logging are not enabled.
```

Comment:
Do both of them always need to be enabled or disabled together? Or can you enable one but not the other? This document doesn't really address that.

Reply:
@bo-quan @dzou-hpe - do either of you know the answer to this question?

Reply:
They don't always need to be enabled or disabled together.